### PR TITLE
upgrade react router to v6

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###

- Route highlighting while completing the default route `/oview`

  
### Release Note ###
   None

### How to test ###
-  start url prefix exp
-  start none prefix exp

### CODE REVIEW

File: ts/webui/src/index.tsx [hunk 1]
```diff
@@ -13,24 +13,24 @@ import * as serviceWorker from './serviceWorker';
...
-        </Suspense>
-    </Router>,
+    <Suspense
+        fallback={
+            <div className='loading'>
+                <img title='loading-graph' src={(path || '') + '/loading.gif'} />
+            </div>
+        }
+    >
+        <Router basename={path === undefined ? '' : path}>
```
Comment (liuzhe-lz) [inline ts/webui/src/index.tsx:23]: `path ?? ''`

**Comment by liuzhe-lz** [comment]
After v2.9 release I plan to bump all dependencies to latest.
It includes nodejs 18, react 18, etc.

**Review by liuzhe-lz — APPROVED** [review]

File: ts/webui/src/index.tsx (hunk 1 above)
Comment (Lijiaoa) [reply ts/webui/src/index.tsx:23]: What's the difference between `??` and `||`

Comment (liuzhe-lz) [reply ts/webui/src/index.tsx:23]: > What's the difference between `??` and `||`

`??` checks `undefined` and `null`, while `||` checks all falsy value (false, zero, empty string, etc).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Comment (Lijiaoa) [reply ts/webui/src/index.tsx:23]: Thanks, get it!

**Review by ultmaster — APPROVED** [review]

**Pull request merged**